substrate/keystore/src/lib.rs

```diff
@@ -14,7 +14,7 @@
 // You should have received a copy of the GNU General Public License
 // along with Polkadot.  If not, see <http://www.gnu.org/licenses/>.
```

Comment:
update doc comment

Reply:
Could you please clarify, what needs to be updated?

Reply:
it says `Polkadot` instead of substrate

Reply:
Well, currently there are a ton of sources in the `substrate` that mention Polkadot in their headers. So, they simply weren't refactored yet?

Reply:
yep, substrate headers should say `Substrate`